postgres.tf

```diff
@@ -40,7 +40,7 @@ resource "null_resource" "bootstrap" {
   }
 
   provisioner "local-exec" {
-    command = "aws lambda invoke --function-name ${aws_lambda_function.db_deploy.arn} --region ${var.region} 'invoke-response.out'"
+    command = "aws lambda invoke --function-name ${aws_lambda_function.db_deploy.arn} --profile ${var.profile} --region ${var.region} 'invoke-response.out'"
```

Comment:
This is kind of a bug-fix. The way this _was_ written, the `default` aws_profile would always be used for this `invoke` command. Since `var.profile` defaults to that `default` value, the simplest case isn't changed. This change also supports the use-case where a non-default profile should be used.